lib/player.rb

```diff
@@ -0,0 +1,14 @@
+class Players
+  def initialize(name, symbol)
+    @name = name
+    @symbol = symbol
+  end
+
+  def description
+    puts "#{@name} has won"
+  end
+
+  def details
+    puts 'Game is a draw'
+  end
+end
```

Comment:
- [ ] [Please fix] No `puts/print` and `gets.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)). In order to fix this you can define the two methods in the `main.rb` file similar to how you defined the `player_move` method. Of course, `description` should accept a name parameter.